src/redux/actions/loginAction.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import $ from "jquery";
+import history from "../../routes/history";
+import {
+  LOGIN_SUCCESSFUL,
+  LOGIN_ERROR,
+  LOGOUT_SUCCESSFUL
+} from "../action_types";
+import homepage from "../../views/Home/index";
+
+const url = "https://ah-magnificent6-staging.herokuapp.com/api/";
+
+export const loginSuccessful = message => ({
+  type: LOGIN_SUCCESSFUL,
+  message
+});
+
+export const loginError = error => ({
+  type: LOGIN_ERROR,
+  error
+});
+
+export const logoutsuccessful = () => ({
```

Comment:
Add comments to your action creators. Comment format

/(asterisk)(asterisk)
 (asterisk) Login successfully
 (asterisk)
 (asterisk) @param {string} message
 (asterisk) @return {object} type and payload
 (asterisk)/

Where there's asterisk please replace with the actual symbol

Reply:
We have implemented the above changes.

Reply:
Done